<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fenMission</class>
 <widget class="QWidget" name="fenMission">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mission</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="6">
    <widget class="QCheckBox" name="cb_Admin">
     <property name="text">
      <string>Mode Admin</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Opérateurs:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Navire:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="le_ChefMission"/>
   </item>
   <item row="8" column="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>51</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Mission:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Zone:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QDateEdit" name="de_Fin"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="le_Mission"/>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Equipements</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="0">
       <widget class="QCheckBox" name="cb_Peskavel">
        <property name="text">
         <string>Peskavel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="cb_Sparker">
        <property name="text">
         <string>Sparker</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QCheckBox" name="cb_Osea">
        <property name="text">
         <string>Osea</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="cb_Klein">
        <property name="text">
         <string>Klein3000</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="cb_2040">
        <property name="text">
         <string>EM2040</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="cb_Scout">
        <property name="text">
         <string>Scout</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="cb_Hydrins">
        <property name="text">
         <string>Hydrins</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="cb_EK60">
        <property name="text">
         <string>EK60</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="cb_Maregraphe">
        <property name="text">
         <string>Maregraphe</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="cb_Roxann">
        <property name="text">
         <string>RoxAnn</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QCheckBox" name="cb_ME70">
        <property name="text">
         <string>ME70</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="6">
    <widget class="QPushButton" name="btn_Annuler">
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Début :</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="btn_Valider">
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Chef de mission:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QTextEdit" name="te_Operateurs">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>91</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Fin :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="le_Zone"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="le_Navire"/>
   </item>
   <item row="1" column="5">
    <widget class="QDateEdit" name="de_Debut"/>
   </item>
   <item row="7" column="1">
    <widget class="QGroupBox" name="gb_Datas">
     <property name="title">
      <string>Données à enregistrer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
